core-plugin/src/com/google/gct/idea/debugger/JavaUtil.java

```diff
@@ -85,4 +87,12 @@ private static String getCloudPathFromFile(@Nullable Project project, @Nullable
     }
     return getCloudPathFromJavaFile((PsiJavaFile)javaFile);
   }
+
+  // Used to fill ourLocationMaps with test data.
+  protected static void initializeLocationsOnUnitTest(
```

Comment:
It does work. I'll go ahead and remove VisibleForTesting too, unless you have any strong objections.